### ISSUE TITLE
运行示例代码，出现如下提示，该怎么处理？

### ISSUE THREAD

**Issue by Li-Lai** [QwenLM/Qwen1.5#24]
2024-02-07 06:36:08,182 - modelscope - INFO - PyTorch version 1.12.1 Found.
2024-02-07 06:36:08,183 - modelscope - INFO - Loading ast index from /root/.cache/modelscope/ast_indexer
2024-02-07 06:36:08,221 - modelscope - INFO - Loading done! Current index file version is 1.12.0, with md5 155281490ae47dfe7d8f4ba91b079bfc and a total number of 964 components indexed
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:151645 for open-end generation.

**Comment by JustinLin610** [comment]
I guess you don't need to worry about the warning. Just something about pad token

**Comment by jklj077** [comment]
closed in favor of https://github.com/QwenLM/Qwen1.5/issues/26

**Comment by 307yin** [comment]
请问是怎么解决的呀 我这个也搞崩好久了
